docs/source/concepts/runner.rst

```diff
@@ -189,8 +189,45 @@ and used in the same service. For example:
 Custom Model Runner
 ^^^^^^^^^^^^^^^^^^^
 
-.. TODO::
-    Document creating custom Runnable with models from BentoML model store
+
+Custom Runnable built with Model from BentoML's model store:
+
+.. code::
+
+    from __future__ import annotations
+
+    from typing import Any
+
+    import bentoml
+    from bentoml.io import JSON
+    from bentoml.io import NumpyNdarray
+    from numpy.typing import ArrayLike
+
+
+    bento_model = bentoml.sklearn.get("spam_detection:latest")
+
+
+    class SpamDetectionRunnable(bentoml.Runnable):
+        SUPPORTS_CPU_MULTI_THREADING = True
+
+        def __init__(self):
+            # load the model instance
+            self.classifier = bento_model.load_model()
```

Comment:
Let's just use this API so we don't have multiple ways to load a model from the bento model?
```suggestion
            self.classifier = bentoml.sklearn.load_model(bento_model)
```